aziotctl/src/internal/check/checks/aziot_version.rs

```diff
@@ -0,0 +1,123 @@
+                return Ok(CheckResult::Ignored);
+            }
+
+            let connector =
+                http_common::MaybeProxyConnector::new(shared.cfg.proxy_uri.clone(), None, &[])
+                    .context("could not initialize HTTP connector")?;
+            let client: hyper::Client<_, hyper::Body> = hyper::Client::builder().build(connector);
+
+            let mut uri: hyper::Uri = "https://aka.ms/latest-aziot-stable-non-lts"
+                .parse()
+                .expect("hard-coded URI cannot fail to parse");
+            let LatestVersions { aziot } = loop {
+                let req = {
+                    let mut req = hyper::Request::new(Default::default());
+                    *req.uri_mut() = uri.clone();
+                    req
+                };
+
+                let res = client.request(req).await.with_context(|| {
+                    format!("could not query {} for latest available version", uri)
+                })?;
+                match res.status() {
+                    status_code if status_code.is_redirection() => {
+                        uri = res
+                            .headers()
+                            .get(hyper::header::LOCATION)
+                            .context("received HTTP redirect response without location header")?
+                            .to_str()
+                            .context(
+                                "received HTTP redirect response with malformed location header",
+                            )?
+                            .parse()
+                            .context(
+                                "received HTTP redirect response with malformed location header",
+                            )?;
+                    }
+
+                    hyper::StatusCode::OK => {
+                        let body = hyper::body::aggregate(res.into_body())
+                            .await
+                            .context("could not read HTTP response")?;
+                        let body = serde_json::from_reader(hyper::body::Buf::reader(body))
+                            .context("could not read HTTP response")?;
+                        break body;
+                    }
+
+                    status_code => {
+                        return Err(anyhow!("received unexpected response {}", status_code))
+                    }
+                }
+            };
+            aziot
+        };
+        self.expected_version = Some(expected_version.clone());
+
+        let actual_version = env!("CARGO_PKG_VERSION");
+        self.actual_version = Some(actual_version.to_owned());
+
+        if expected_version != actual_version {
+            return Ok(CheckResult::Warning(
+                anyhow!(
+                    "Installed aziot-identity-service package has version {} but {} is the latest stable version available.\n\
+                    Please see https://aka.ms/iotedge-update-runtime for update instructions.",
```

Comment:
@micahl Do we have any docs planned for this? I don't believe we do.

If not, I can proactively create an `https://aka.ms/aziot-update-runtime` that points to the same page as this one for now, but gives us the flexibility to make it point to something else in the future.

Reply:
@arsing correct, we don't have something for this yet.  Please create a redirect link that we can update later.  Also see my separate email on the difference between fwlink and aka.ms.

Reply:
Okay, I've created the new aka.ms URL.

re: fwlink - We use target links without the language code, eg this one redirects to `https://docs.microsoft.com/azure/iot-edge/how-to-update-iot-edge`, not `https://docs.microsoft.com/en-us/azure/iot-edge/how-to-update-iot-edge`. The former redirects to the latter or to the appropriate language based on user agent's `accept-language` header. So eg a user whose browser sends `accept-language: it-it` will get redirected to the `/it-it/` page. So I think it's fine in that regard.

At any rate, I don't know how to fill the required fields it wants me to fill for a fwlink URL, and if we really want to switch to fwlink we have a bunch of these links to update (both in this repo and in iotedge), so let's do that after GA.